MAINTAINERS.md

```diff
@@ -1,17 +1,166 @@
+
+[permissions]: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role
+
+|                                                                                                                         _Responsibilities_  |       Contributor       |       Code Owner        |       Maintainer        |
+| :------------------------------------------------------------------------------------------------------------------------------------------ | :---------------------: | :---------------------: | :---------------------: |
+|                                                                                                                                             |                         |                         |                         |
+| Follow the Code of Conduct                                                                                                                  |            ✓            |            ✓           |            ✓            |
+| Follow Contribution Guidelines                                                                                                              |            ✓            |            ✓           |            ✓            |
+| Ensure Contribution Guidelines are followed                                                                                                 |            ✗            |            ✓           |            ✓            |
+| Cleanup Stale PRs and Branches impacting a component                                                                                        |            ✗            |            ✓           |            ✓            |
+| Co-own component or aspect of the library,<br>  including contributing: bug fixes, implementing features,<br> and performance optimizations |            ✗            |            ✓           |            ✓            |
+| Co-own on technical direction of component or<br> aspect of the library                                                                     |            ✗            |            ✓           |            ✓            |
+| Co-own the project as a whole,<br> including determining strategy and policy for the project                                                |            ✗            |            ✗           |            ✓            |
+|                     |                         |                        |                         |
+|                                                                                                                         **_Privileges_**    |     **Contributor**     |     **Code Owner**      |     **Maintainer**      |
+|                                                                                                                         Permission granted  |   [Write][permissions]   |   [Write][permissions]  | [Maintain][permissions] |
```

Comment:
Two comments regarding the permissions.

First, the table says contributors have write permissions, but the detailed section for contributors mentions only read permissions.

Second, if contributors have write permissions, does it mean that this is the default permission level for anybody? Or does this assume some "process to become a contributor" that grants additional permissions?

Reply:
I agree with the previous discussions that we've had about Contributors being able to create branches in the repo instead of working through forks until they're a Code Owner, and that requires Write access. I think for now this should be the default for anyone and there's no additional process to become a contributor.

Reply:
I've updated the table and the wording in Contributor privileges. Please take a look and let me know if you have any other suggestions for the wording.